edx_event_bus_kafka/internal/tests/test_producer.py

```diff
@@ -342,3 +344,34 @@ def test_serialize_and_produce_to_same_topic(self, mock_context):
             # headers are tested elsewhere, we just want to verify the topics
             headers=ANY,
         )
+
+
+class TestCommand(TestCase):
+    """
+    Test produce_event management command
+    """
+    @patch('edx_event_bus_kafka.management.commands.produce_event.logger')
+    @override_settings(EVENT_BUS_TOPIC_PREFIX='dev')
+    @override_settings(EVENT_BUS_KAFKA_SCHEMA_REGISTRY_URL='http://localhost:12345')
+    @override_settings(EVENT_BUS_KAFKA_BOOTSTRAP_SERVERS='localhost:54321')
```

Comment:
[nit] This might feels a bit cleaner:

```suggestion
    @override_settings(
      EVENT_BUS_TOPIC_PREFIX='dev',
      EVENT_BUS_KAFKA_SCHEMA_REGISTRY_URL='http://localhost:12345',
      EVENT_BUS_KAFKA_BOOTSTRAP_SERVERS='localhost:54321',
    )
    @patch('edx_event_bus_kafka.management.commands.produce_event.logger')
```

Reply:
(or I guess with a deeper indent -- pycodestyle would probably yell)